Pebble Station

Kromabae opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Small map, maybe like Saltern size but smaller.

Idea behind this map was to have more smaller options for lower pop crews as to not have too much empty space between player interactions.
This is mostly intended for Salamander because people mostly chill there and hang out, and I thought that I'd give a shot at making a map for lower pop in between 0-35ish. Map has most emphasis on the common area for a more social hangout for smaller crews.

I know there's a map freeze, but this is a very very small map and I believe it's desperately needed for the current small pop like on Salamander.

It's been play tested on with around 5-7 people and it's received good praise so far.

**Screenshots**

![pebble-0](https://user-images.githubusercontent.com/97402700/192169706-aca91ed7-1f18-42dc-a269-8f542e1a4032.png)

<!-- If applicable, add screenshots to showcase your PR. If your PR is a visual change, add
screenshots or it's liable to be closed by maintainers. -->

**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.

For writing actual entries, don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers
-->

:cl: Kroma
- add: Add Pebble, a new small pop map for those smaller pops.

mirrorcult commented:
didnt we already get a new lowpop with omega

Kromabae commented:
Omega is bigger than Saltern, this is smaller than Saltern

moonheart08 commented:
Unless you can convince a few maintainers to make an exception, a map needs derotated to add this.
It does look quality though.

moonheart08 commented on Resources/Prototypes/Maps/pebble.yml:
```diff
@@ -0,0 +1,47 @@
+  mapName: 'Pebble Station'
+  mapPath: /Maps/pebble.yml
+  minPlayers: 0
+  maxPlayers: 35
```
```suggestion
  maxPlayers: 20
```

Kromabae replied:
Fixed, thanks

moonheart08 commented on Resources/Prototypes/Maps/pebble.yml:
```diff
@@ -0,0 +1,47 @@
+      mapNameTemplate: '{0} Pebble Station {1}'
+      nameGenerator:
+        !type:NanotrasenNameGenerator
+        prefixCreator: 'TG'
```
```suggestion
        prefixCreator: '14'
```

Kromabae replied:
Changed, thanks

Emisse commented:
feel free to derotate a map if you wish

EmoGarbage404 commented:
Just from looking at the rendering provided, i have a few immediate gripes.

Firstly, this map has no maintenance tunnels. This is already a longstanding issue with a lot of maps lacking extensive maintenance areas, but this map has probably as close to 0 maintenance as you could feasibly get. As a result, its going to have a lot less emergency supplies and maintenance closets. And while i can see that this is partially remedied with some wall mounted emergency and fire closets, I still think it would benefit more from some side areas and more closets.

My second major issue with this map is its inefficient use of space and the frankly cramped layout in most areas. I understand the intent is a lowpop map, but i feel like some of these areas truly have 0 space in them. Medical is split in half by a lobby that could honestly be half the size. Science is weirdly chunked by a useless robotics room (also the virology room should probably be in med). Botany also has a hydrobe tucked behind a table and is so cramped you have to walk over the trays.

This is in contrast with the other strangely spacious areas on the map. All heads not only have a full office, but ALSO a full bedroom AS WELL AS a bathroom, which feels excessive. In a similar vein, atmospherics is the size of the rest of engineering and science combined, and has a lot of very empty areas in it.

In terms of the structural setup, I feel that there is a definite overuse of reinforced walls and reinforced windows. Also, there's occasional diagonal windows along with firelocks over windows, which is kind of sinful.

The last thing i'll touch on is the job division. The way it's currently set up is basically a single copy of every job available. I would offer that you reconsider this and perhaps remove some of the more non-essential jobs (detective, musician, chemist?) and perhaps try to increase the number of jobs in some of the other departments, at least in terms of late-join slots. 

Also, i know moony suggested a max player count of 20, but there are 35~ jobs available, which means this station is never going to be running at full crew capacity.

Kromabae commented:
Changes have been made from feedback received, thanks everyone for the feedback. 
Edited new pic in original post.

some of the feedback i didn't want to change. I feel like the map for what it is and what it's designed for is enough. although i can see the need for more maints, i personally see this more of a map for lower pop and will probably fit right in for the current player pop of salamander.

replaced a few reinforced walls with normal walls in certain areas.

made atmos smaller and more basic.

fixed some oversights.

Again thanks for the feedback and I shall take that into consideration if i ever do another map.

Kromabae commented:
I'm just going to go back on the drawing board it seems.....
